python_modules/dagster/dagster/_grpc/types.py

```diff
@@ -238,23 +240,28 @@ def __new__(
             ),
         )
 
+    def _get_compressed_args(self) -> str:
+        # Compress, then base64 encode so we can pass it around as a str
+        return base64.b64encode(zlib.compress(serialize_dagster_namedtuple(self).encode())).decode()
```

Comment:
Re-requesting review bc I had to add base64 encoding to be able to pass the compressed bytes as a string in the CLI command or env var